radio/src/targets/common/arm/stm32/serial2_driver.cpp

```diff
@@ -101,6 +101,8 @@ void serial2Init(unsigned int mode, unsigned int protocol)
         uart3Setup(FRSKY_D_BAUDRATE, true);
       }
       break;
+    case UART_MODE_LUA:
+      uart3Setup(DEBUG_BAUDRATE, false);
```

Comment:
Can this not be DEBUG_BAUDRATE because 115k inverted requires special hardware to read. Softwareserial works well on pretty much all Arduino hardware at 57k and is able to read inverted logic. 

If you can, change it to 57k and I don't think anyone would complain or make it some config option.

Thanks!